test/components/stage_help_test.js

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import sinon from "sinon"
+import { shallow } from "enzyme"
+
+import { StageHelp } from "../../web/static/js/components/stage_help"
+
+describe("<StageHelp />", () => {
+  let wrapper
+  let retro
+
+  // Setup JSDom so that react can inject a portal
+  const iconRoot = global.document.createElement("div")
+  iconRoot.setAttribute("id", "stage-help-icon")
+  const body = global.document.querySelector("body")
+  body.appendChild(iconRoot)
```

Comment:
dude, nice work on these tests. have to imagine it took some digging to get the setup right.

Reply:
The internet knows all.